lib/CloudinaryPlugin.js

```diff
@@ -98,20 +97,16 @@ class CloudinaryPlugin {
       }
     ];
 
-    this.config = givenConfig;
+    this.config = config;
 
     const { NODE_ENV } = process.env;
-    const missingCredMessage = 'Cloudinary Plugin cannot be used without the ApiKey, the ApiSecret and the CloudName of your Cloudinary Account';
-
     const { cloudinaryCloudName, cloudinaryApiKey, cloudinaryApiSecret } = this.config;
```

Comment:
Can i access the running version of Kuzzle ? So it can use Vault feature if the version is higher than 1.8.1 and use the config value if not. This will widen the compatibility of the plugin.

Reply:
I believe that you have `this.config.version` (to be confirmed, this is from the top of my head)

Reply:
Yes you're right. I'll push the changes soon :) Thanks

Reply:
Ok i added the vault feature and restricted to higher version of Kuzzle. (PR #2 ) It will be fully working from version 1.8.2 and higher since i'm affected with the [long key encryption issue](https://github.com/kuzzleio/kuzzle/pull/1326)